### DISCUSSION TITLE
postcss config not recognized in CLI command

### DISCUSSION THREAD

**Discussion by kpettinga** [tailwindlabs/tailwindcss#11657] [Help]
**The Problem:**
When I run tailwind from the command line with the `--post-css` option included, the configuration specified in postcss.config.js does not seem to be recognized. This is evident to me because I am using the `postcss-import-ext-glob` package to handle glob imports in my css but Tailwind is not compiling the glob pattern I specify. It simply outputs the `@import-glob` statement that I wrote in my source css: 
```css
/* {...compiled css...} */

@import-glob 'components/**/*.css';
```

**Expected behavior**
I expected the glob to be parsed out into individual import statements for each .css file found within the `components` directory. Then, the css from each of those imports should have been included and compiled in my final output css.

**Other notes**
I also notice that if I specify a bad postcss config file (e.g. `--post-css non-existent-file.js`), there is no error thrown. This furthers my suspicion that the --post-css option is not actually looking for the specified config file.

**This is the the command I am using to build my CSS:**
`npx tailwind -i css/app.css -o public/dist/app.css --post-css postcss.config.js`

**This is my postcss.config.js**
```js
module.exports = {
	minify: true,
	plugins: {
		'postcss-import-ext-glob': {},
		'postcss-import': {},
		'tailwindcss/nesting': 'postcss-nesting',
		tailwindcss: {},
		autoprefixer: {},
	}
}
```

**This is my tailwind.config.js**
```js
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ['./components/**/*.{html,twig}'],
  theme: {
    extend: {},
  },
  plugins: [],
}
```

**Relevant package versions:**
```js
devDependencies: {
  "autoprefixer": "^10.4.14",
  "postcss": "^8.4.26",
  "postcss-import": "^15.1.0",
  "postcss-import-ext-glob": "^2.1.1",
  "tailwindcss": "^3.3.3",
}
```

**Comment by wongjn** [comment]
To specify a custom PostCSS configuration, consider using the `--postcss` flag.

**Reply by kpettinga** [reply]
Thank you spotting my typo.... 🤦
